packages/manager/modules/pci/src/projects/project/notebooks/components/configuration-command/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,6 @@
+{
+  "pci_notebook_configuration_command_title": "Création automatique d’un notebook équivalent",
+  "pci_notebook_configuration_command_description": "Vous pouvez créer le même notebook en utilisant ces lignes de commande dans votre ovhai CLI.",
+  "pci_notebook_configuration_command_help": "Pour plus d’informations, lisez la documentation sur le CLI ovhai.",
```

Comment:
i am note totaly sure but i think we must use "la CLI ovhai" not "le CLI ovhai."
```suggestion
  "pci_notebook_configuration_command_help": "Pour plus d’informations, lisez la documentation sur la CLI ovhai.",
```

Reply:
@ganeshkumar1989 , we have a confirmation.
We must use:
```suggestion
  "pci_notebook_configuration_command_help": "Pour plus d’informations, lisez la documentation sur la CLI ovhai.",
```

Reply:
This is fixed. It is `La CLI`...
Thank you for the suggestion :)